exporter/zipkin/zipkin.go

```diff
@@ -38,25 +47,50 @@ type Exporter struct {
 // can be created with the openzipkin library, using one of the packages under
 // github.com/openzipkin/zipkin-go/reporter.
 //
-// localEndpoint sets the local endpoint of exported spans.  It can be
+// localEndpoint sets the default local endpoint of exported spans.  It can be
 // constructed with github.com/openzipkin/zipkin-go.NewEndpoint, e.g.:
 // 	localEndpoint, err := NewEndpoint("my server", listener.Addr().String())
 // localEndpoint can be nil.
+//
+// localEndpoint can be overridden on a per-span basis by setting Span attributes
```

Comment:
This doesn't look like a Zipkin-only case. Can't we implement it more generally?

We can export a canonical attribute key from the trace package for the service name.

Most importantly, it needs to be reflected in the specs especially if we will preserve the opencensus prefix to OpenCensus libraries.